lib/logger/lib/logger.ex

```diff
@@ -19,9 +19,12 @@ defmodule Logger do
     * Formats and truncates messages on the client
       to avoid clogging `Logger` handlers.
 
-    * Alternates between sync and async modes to remain
-      performant when required but also apply back-pressure
-      when under stress.
+    * Provides multiple forms of [overload protection](https://www.erlang.org/doc/apps/kernel/logger_chapter.html#protecting-the-handler-from-overload):
```

Comment:
The line is too long, unsure how to break it up or if there's a better way to link to it - but I found calling out the specific chapter for details valuable.

Reply:
It's fine, no worries!